gsk4/src/text_node.rs

```diff
@@ -75,6 +75,7 @@ impl TextNode {
     //
     // Due to https://gitlab.gnome.org/GNOME/gtk/-/issues/3675 this function wasn't properly
     // exported in 4.0.0 and so requires 4.1.2 at least for it to work properly.
+    #[doc(alias = "gsk_text_node_has_color_glyphs")]
```

Comment:
there's one already, it's just before the comment

Reply:
Then it's a bug! I wondered If I should include comments in the "look back", well, that confirms it.

Reply:
Fixed in https://github.com/gtk-rs/checker/pull/22.